sections/rust.zsh

```diff
@@ -19,19 +19,34 @@ SPACESHIP_RUST_COLOR="${SPACESHIP_RUST_COLOR="red"}"
 # ------------------------------------------------------------------------------
 
 # Show current version of Rust
-spaceship_rust() {
+spaceship_async_job_load_rust() {
   [[ $SPACESHIP_RUST_SHOW == false ]] && return
 
+    async_job spaceship spaceship_async_job_rust $PWD
+}
+
+spaceship_async_job_rust() {
   # If there are Rust-specific files in current directory
+  setopt extendedglob
   [[ -f Cargo.toml || -n *.rs(#qN^/) ]] || return
 
   spaceship::exists rustc || return
 
   local rust_version=$(rustc --version | grep --color=never -oE '[[:digit:]]+\.[[:digit:]]+\.[[:digit:]]')
 
+  [[ -z $rust_version ]] && return
+
+  [[ "${rust_version}" =~ ^[0-9].+$ ]] && rust_version="v${rust_version}"
```

Comment:
Won't `rust_version` always start with a digit, because of the `grep` in the line 35?